contrib/vector_demo.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+# This script assumes the ASTE binaries and python scripts are in $PATH
+# Furthermore, execute in ./contrib or copy precice.xml from there to $PWD
+
+
+# Download the red blood cell
+test -f rbc.vtk || wget "https://people.sc.fsu.edu/~jburkardt/data/vtk/rbc_001.vtk" -O rbc.vtk
+
+# Download the bunny
+test -f bunny.vtk || wget "https://www.ece.lsu.edu/xinli/Meshing/Data/bunny.vtk" -O bunny.vtk
+
+# Evaluate the function x+y on the bunny, write to colored.vtk
+eval_mesh.py bunny.vtk -o colored.vtk "x + y" fdim 3
```

Comment:
The provided function `"x + y"` evaluates to a scalar, but should evaluate to a 3-dimensional vector.